tests/jet_test.py

```diff
@@ -303,6 +303,8 @@ def test_cumprod(self):    self.unary_check(jnp.cumprod)
   def test_cummax(self):     self.unary_check(partial(lax.cummax, axis=0))
   @jtu.skip_on_devices("tpu")
   def test_cummin(self):     self.unary_check(partial(lax.cummin, axis=0))
+  @jtu.skip_on_devices("tpu")
+  def test_dynamic_slice(self):     self.unary_check(partial(lax.dynamic_slice, start_indices=(0,0), slice_sizes=(1,1)))
```

Comment:
The values for `start_indices` and `slice_sizes` are arbitrary.

Also: 
The original motivation was to make `jet` work on `numpy.roll`. But since the missing key was `dynamic_slice`, I tested `dynamic_slice`, and not `numpy.roll`. Please let me know if I should change this test. (Feel invited to resolve this conversation if everything is fine.)